Src/Plugins/Security/SecurePlaintextSecretsValidators/GitHubTokenValidationHelper.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk;
+
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
```

Comment:
>Security [](http://example.com/codeflow?start=62&length=8)

Exactly same as GitHubTokenValidationHelper.cs in Security.Internal?

Reply:
almost same except `GitHubEndpoints` property. pls see my comment about it. The internal plugin can use this class and override the `GitHubEndpoints` if needed.